src/index.html

```diff
@@ -1,13 +1,374 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Miami</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page__body">
+    <header class="page__section header">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-actions">
+            <a href="#menu" class="icon icon--menu"></a>
+            <a href="/" class="logo">
+              <img
+                class="logo__image"
+                src="./images/logo.png"
+                alt="logo"
+              >
+            </a>
+            <a href="tel: +1 234 555-55-55" class="icon icon--phone header__call"></a>
+          </div>
+          <h1 class="header__title">
+            Miami<br>
+            Condo Kings
+          </h1>
+        </div>
+      </div>
+    </header>
+
+    <nav class="menu page__menu" id="menu">
+      <div class="container">
+        <div class="menu__content">
+          <div class="top-actions menu__top">
+            <a href="#" class="icon icon--cross"></a>
+            <a href="/" class="logo">
+              <img
+                class="logo__image"
+                src="./images/logo.png"
+                alt="logo"
+              >
+            </a>
+          </div>
+          <ul class="menu__list">
+            <li class="menu__item">
+              <a href="#" class="menu__link">Home</a>
+            </li>
+            <li class="menu__item">
+              <a href="#about-us" class="menu__link">About us</a>
+            </li>
+            <li class="menu__item">
+              <a href="#gallery" class="menu__link">Gallery</a>
+            </li>
+            <li class="menu__item">
+              <a href="#testimonials" class="menu__link">Testimonials</a>
+            </li>
+            <li class="menu__item">
+              <a href="#contact-us" class="menu__link">Contact us</a>
+            </li>
+          </ul>
+
+          <a
+            href="tel:+1 234 555-55-55"
+            class="menu__call menu__call--phone"
+          >+1 234 555-55-55</a>
+          <a
+            href="tel:+1 234 555-55-55"
+            class="menu__call menu__call--action"
+          >Call to order</a>
+        </div>
```

Comment:
elements with number and `call to order` are a bit offcentered
![image](https://user-images.githubusercontent.com/37161315/170692170-7a95b96a-2ef7-48d1-958f-80f3ba5aa3db.png)